atsign-foundation/mwc_demo
# feat: mostly mvp level complete

New, simplified data model. Persistence working and the UI component flow first pass completed.



**- What I did**
* Refactored to a new, simplified data model. 
* Persistence working and the UI component flow first pass completed.

**- How I did it**

**- How to verify it**
* run the app
* add a device
* check the devices screen to see the device
* add a receiver after selecting a device

**- Description for the changelog**

feat: Refactored to a new, simplified data model. Persistence is now working and the UI component flow first pass completed.
